### DISCUSSION TITLE
Coupled transmission lines

### DISCUSSION THREAD

**Discussion by DanielSank** [ra3xdh/qucs_s#785] [Q&A] [answered]
I am interested in circuits with coupled transmission lines. I find https://github.com/ra3xdh/qucsator_rf/blob/7494604ceab49ad18c5e25390078479add4594c1/src/components/ctline.cpp which has a model for coupled transmission lines. However, I do not see this model in the GUI when building a schematic.

How would I access this model in the schematic?

**Comment by ra3xdh** [comment, marked as answer]
Everything is simple. You are simulating in Ngspice mode. The microstrip devices are not compatible with Ngspice and hidden to prevent its insertion in the schematic by mistake. You have to switch to Qucsator mode using the drop-down list in the toolbar. Make sure you are using the version >= 24.2.0 It is shipped with QucsatorRF by default. 

![image](https://github.com/ra3xdh/qucs_s/assets/4920080/4192f8f8-311e-426b-8939-c639b58cb289)

**Comment by ra3xdh** [comment]
I have transferred this issue to discussions/Q&A https://github.com/ra3xdh/qucs_s/discussions You may ask the questions not related to bug reports here.

**Comment by ra3xdh** [comment]
@DanielSank the following resources about Qucs-S may be helpful for you:

* Qucs-S tutorial https://ra3xdh.github.io/pdf/qucs_s_tutorial.pdf Information is mostly up to date. The chapters 8 and 9 explain about RF simulaiton and simulator switching. Since the v24.2.0 QucsatorRF is shipped inside the package and no need to compile it manually. 
* Video tutorial from Kasper Nielsen: https://www.youtube.com/watch?v=90RaVy38DB8
* Advanced simulation guide: https://habr.com/ru/articles/789452/ Sorry, only Russian version is available.
